dev-docs/bidders/relay.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: bidder
+title: Relay
+description: Prebid Relay Bidder Adapter
+biddercode: relay
+media_types: banner, video, native
+pbjs: true
+pbs: false
+safeframes_ok: true
+floors_supported: true
+fpd_supported: true
```

Comment:
In order to claim FPD support, you need to have a section in the document below that describes how you support it. e.g. site.content.data, site.ext.data, user.data, user.ext.data

Reply:
Does FPD support have a different meaning than "usage of the `ortb2`" module are supported?

In our case there are not specific keys/values during ad unit configuration that we are looking for that need to be placed on the Request to us, rather if the publisher needs to modify the outgoing OpenRTB BidRequest to us in any way they can use the `ortb2` module to do whatever they need and we will support that.

Reply:
@bretg Do you have advisement? Does my response satisfy or does the definition convey a different meaning and the preference would be to set `fpd_supported: false` ?

Reply:
@bretg Just pinging again about this previous question:

> Do you have advisement? Does my response satisfy or does the definition convey a different meaning and the preference would be to set `fpd_supported: false` ?

Reply:
@jcswart - if you support all fields of first party data, just say so. "We support all FPD fields: site.content.data, site.ext.data.*, user.data, user.ext.data.*"